planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1483,7 +1434,9 @@ bool NormalLaneChange::getLaneChangePaths(
       const auto prepare_length = utils::lane_change::calcPhaseLength(
         current_velocity, getCommonParam().max_vel, longitudinal_acc_on_prepare, prepare_duration);
 
-      const auto ego_dist_to_terminal_start = dist_to_end_of_current_lanes - lane_change_buffer;
+      const auto ego_dist_to_terminal_start =
+        common_data_ptr_->transient_data.dist_from_ego_to_current_terminal_end -
+        common_data_ptr_->transient_data.current_dist_buffer.min;
```

Comment:
isn't this same as `common_data_ptr_->transient_data.dist_from_ego_to_current_terminal_start`

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/8849/commits/56c45d8960f7574105c07973019378729f833fab